First stab at sponsors page #11

wpp opened the pull request:
Hi @tgxworld I had some time and saw #11. Thought I'd give it a shot. Feel free to criticise and nitpick (texts putting stuff in locale etc...) ! I'll try and update as soon as I can.

tgxworld commented on app/views/static_pages/sponsors.html.haml:
```diff
@@ -0,0 +1,18 @@
+    .row.sponsor-row
+      .col-xs-5
+        %div{class: t("sponsors.#{sponsor}.class_name")}
+          %span.l-text-hidden t("sponsors.#{sponsor}.name")
```
missing a ```=```here.

tgxworld commented on app/views/layouts/_footer.html.haml:
```diff
@@ -1,32 +1,9 @@
   %h5
     Built and maintained by
-    = link_to '@tgxworld', 'https://github.com/tgxworld', target: '_blank'
+    = link_to '@tgxworld.', 'https://github.com/tgxworld', target: '_blank'
```
I think the periods shouldn't be part of the links. You can use ```succeed '.' do``` :smile_cat:

tgxworld commented on app/views/layouts/_footer.html.haml:
```diff
@@ -1,32 +1,9 @@
+    = link_to '@tgxworld.', 'https://github.com/tgxworld', target: '_blank'
+
+    Many thanks to our
+    = link_to 'sponsors.', sponsors_path, target: '_blank'
```
Ditto https://github.com/ruby-bench/ruby-bench-web/pull/23/files#r24718561

tgxworld commented on app/views/layouts/_footer.html.haml:
```diff
@@ -1,32 +1,9 @@
-          %i= t('footer.bare_metal_server')
-
-        .sponsor-ruby-tune
-          %span.l-text-hidden RubyTune
```
I think we should keep this. Since it appears at the footer, we give the sponsors a higher chance at exposure. My intention for the dedicated sponsors page was to help provide more information about each sponsor.

@SamSaffron Any thoughts?

tgxworld replied:
Hmm but then again a dedicated page only adds about a line more of information.

tgxworld commented on app/views/static_pages/sponsors.html.haml:
```diff
@@ -0,0 +1,18 @@
+
+  %hr
+
+  %h3 Many thanks to our sponors!
```
Typo here. You could also add this into `I18n` :smile:

tgxworld commented on app/views/static_pages/sponsors.html.haml:
```diff
@@ -0,0 +1,18 @@
+      .col-xs-7
+        %h3
+          = t("sponsors.#{sponsor}.name")
+        = link_to t("sponsors.#{sponsor}.link"), target: '_blank' do
```
I think we should make the whole `div` for each sponsor (logo, title text) a link. It isn't immediately clear to me that the description leads to the sponsor's page. :smile_cat:

tgxworld commented:
@wpp Nice work! :smile: I'm more keen on keeping the logos in the footer but adding another link in the `navigation` bar to lead to the sponsors. From the height of the page now, people probably won't be scrolling down much so I think it's good if we can add a link on top.

wpp commented:
Hi @tgxworld,

Thanks for the quick feedback.
I'm not 100% if you wanted me to re-add the footer or not (I did now).
Besides that, I think I've addressed all your remarks.

tgxworld commented:
@wpp Sorry just one more nitpick. Could we set the text to be `vertical-align: middle`? Thanks!

![screenshot from 2015-02-16 23 52 56](https://cloud.githubusercontent.com/assets/4335742/6214902/095a8a52-b637-11e4-8372-c25d4f1fc0b5.png)

wpp commented:
Hi @tgxworld I'm basically this guy:

![css](http://2.bp.blogspot.com/-41v6n3Vaf5s/UeRN_XJ0keI/AAAAAAAAN2Y/YxIHhddGiaw/s1600/css.gif)

somehow that `vertical-align` does not (hing) work as expected.

```css
.sponsor-row {
  margin-top: 40px;
  margin-bottom: 40px;

  .col-xs-7 {
    vertical-align: middle;
  }
```

Could you comment directly in the code if you know how to do this? (Or merge and do it yourself, which might be quicker :))

tgxworld commented:
@wpp  Sorry for taking so long to get to this. Merged in https://github.com/ruby-bench/ruby-bench-web/commit/093c6bf501d78d31d1bb07b906640d32984ed6a3. I didn't manage to get vertical align to work as well. :stuck_out_tongue_closed_eyes:

wpp commented:
@tgxworld no problem :smile:. CSS is just too hard.
